doc/src/manual/noteworthy-differences.md

```diff
@@ -205,8 +205,8 @@ For users coming to Julia from R, these are some noteworthy differences:
     be reversed in Julia relative to NumPy (see relevant section of [Performance Tips](@ref man-performance-tips)).
   * Julia's updating operators (e.g. `+=`, `-=`, ...) are *not in-place* whereas NumPy's are. This
     means `A = ones(4); B = A; B += 3` doesn't change values in `A`, it rather rebinds the name `B`
-    to the result of the right- hand side `B = B + 3`, which is a new array. Use `B[:] += 3`, explicit
-    loops, or `InplaceOps.jl`.
+    to the result of the right- hand side `B = B + 3`, which is a new array. Use `B .+= 3`
```

Comment:
While modifying these lines, should the space exist in "right- hand"? Also might be nice to extend "Use `B .+= 3`..." to "For in-place operation, use `B .+= 3`..."?